<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionEdDlg</class>
 <widget class="QDialog" name="QuestionEdDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Question editing</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="lbCaption">
     <property name="text">
      <string>Question content</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QToolButton" name="tbSelectPicture">
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>20</y>
           <width>32</width>
           <height>32</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Load image from file</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
        <widget class="QToolButton" name="tbDeletePicture">
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>60</y>
           <width>32</width>
           <height>32</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Delete picture</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="fingram.qrc">
           <normaloff>:/resources/delete.png</normaloff>:/resources/delete.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbImage">
        <property name="minimumSize">
         <size>
          <width>330</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>330</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background : rgba(80,80,0,30);
color :  rgba(60,60,0,20);
font-family: sans-serif;
font-size: 20px; 
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>No picture</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QToolButton" name="tbAdd">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>4</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Insert answer</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="fingram.qrc">
        <normaloff>:/resources/add.png</normaloff>:/resources/add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="tbDel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>4</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Delete answer</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="fingram.qrc">
        <normaloff>:/resources/delete.png</normaloff>:/resources/delete.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="tbSort">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>4</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Sort answer</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="fingram.qrc">
        <normaloff>:/resources/sort.png</normaloff>:/resources/sort.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>160</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="fingram.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QuestionEdDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QuestionEdDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
